phpBB/phpbb/template/twig/node/event.php

```diff
@@ -42,40 +46,49 @@ public function compile(\Twig\Compiler $compiler)
 
 		$location = $this->listener_directory . $this->getNode('expr')->getAttribute('name');
 
+		$template_events = [];
+
+		// Group and sort extension template events in according to their priority (0 by default if not set)
 		foreach ($this->environment->get_phpbb_extensions() as $ext_namespace => $ext_path)
 		{
 			$ext_namespace = str_replace('/', '_', $ext_namespace);
+			$priority_key = $this->template_event_priority_array[$ext_namespace][$location] ?? 0;
+			$template_events[$priority_key][] = $ext_namespace;
```

Comment:
I'm not sure if `$template_events` is the best way here since it technically should only be a single event and we're specifying the order this event is called depending on priority.